node.js

```diff
@@ -80,17 +80,22 @@ exports.formatters.o = function(v) {
  */
 
 function formatArgs() {
-  var args = arguments;
+  var args = [];
+  var len = arguments.length;
+  var i = 0;
   var useColors = this.useColors;
   var name = this.namespace;
+  for ( ; i < len; i++) {
```

Comment:
Even though I don't maintain the project, I would still like to hear why `i` is declared above and not here :)